root/pom.xml

```diff
@@ -45,6 +45,11 @@
 				</exclusion>
```

Comment:
Not related to this file, but can you change the Pull Request's name to be descriptive